tests/integration/test_stages.py

```diff
@@ -89,6 +89,10 @@ class CommonStageTest(StageTestBase):
         (datetime.datetime(2014, 6, 22), None, datetime.datetime(2015, 6, 12), "due Jun 12 2015"),
         # after close - should be closed June 12
         (datetime.datetime(2015, 6, 13), None, datetime.datetime(2015, 6, 12), "closed on Jun 12"),
+        # At close date
+        (datetime.datetime(2015, 6, 13), datetime.datetime(2015, 6, 11), datetime.datetime(2015, 6, 13), "due Jun 13"),
+        # Single day stage
+        (datetime.datetime(2015, 6, 13), datetime.datetime(2015, 6, 13), datetime.datetime(2015, 6, 13), "due Jun 13")
```

Comment:
@jbzdak what's the difference between the two test cases?

Ideally, this should contain "precise" check, i.e. 2015-06-13 23:59:59 is still "due", but 2015-06-14 00:00:00 is "closed".